integration/simulation/utils.go

```diff
@@ -107,8 +109,8 @@ func getTransaction(client obscuroclient.Client, txHash common.Hash) *nodecommon
 
 	// We check that there is a valid receipt for each transaction, as a sanity-check.
 	txReceiptJSONMap := getTransactionReceipt(client, txHash)
-	// TODO - Fix issue whereby all receipts have a status of 0 (failure). Then check the status here instead.
-	if txReceiptJSONMap[jsonKeyTxHash] != txHash.Hex() {
+	// Per Geth's rules, a receipt is invalid if the root is not set and the status is non-zero.
```

Comment:
/ Per Geth's rules, a receipt is invalid if the root is not set and the status is zero.

or  , a receipt is valid if:
- status ==1 OR (root.len ==32)